test/define-test.js

```diff
@@ -17,3 +17,85 @@ QUnit.test("Can define stuff", function(assert) {
   let faves = new Faves();
   assert.equal(faves.color, "blue", "Got the value");
 });
+
+QUnit.test("Can not define stuff", function(assert) {
```

Comment:
What is this test testing?

Reply:
It's testing that it doesn't throw if you done have a `static define`.

Reply:
There's an existing test for that: https://github.com/canjs/can-define-class/blob/bf83944ef3499f64e9e01974a30e468af8505be9/test/define-class-test.js#L143-L146

Reply:
Ok, I'll look into what the difference is. In my case, `this.define` is undefined here: https://github.com/canjs/can-define-class/blob/bf83944ef3499f64e9e01974a30e468af8505be9/can-define-class.js#L71.